packages/social-trader/src/components/notifications/components/notification/notification.tsx

```diff
@@ -38,32 +28,46 @@ import {
   $paddingXxsmall
 } from "utils/style/sizes";
 
-enum TYPE {
-  PROFILE = "profile",
-  PLATFORM = "platform"
+import { getColor, getLink, getStaticIconUrl } from "./notification-helpers";
+
+export enum TYPE {
```

Comment:
What happens if the backend changes values of the `type` field in NotificationViewModel?
Maybe this enum should be declared in the API?

Reply:
Backend said me there is no chance to use enum, because mobile app crashes.